src/xtgeo/xyz/_xyz_roxapi.py

```diff
@@ -56,8 +56,10 @@ def _check_input_and_version_requirement(
 
     if attributes is not None:
         if mode == "get" and not isinstance(attributes, (list, bool)):
+            roxutils.safe_close()
```

Comment:
I think we can remove all these `safe_close` from here and just have one at the end of the test `test_check_presence_in_project_errors` to make the test run without the error of "can't reopen project".